ru/turing/courses/lesson2/animals/bird.java

```diff
@@ -0,0 +1,19 @@
+package ru.turing.courses.lesson2.animals;
+
+public class Bird extends Animal {
+    private boolean migration;
+    //true - letit na south,false-no
+    private String singing;
+
+    public bird(String vid, boolean eatingmeat, int age, boolean wild, boolean migration, String singing) {
```

Comment:
```suggestion
    public Bird(String vid, boolean eatingmeat, int age, boolean wild, boolean migration, String singing) {
```
Конструкторы называются точно так же, как класс, а класс и его файл именуются с заглавной буквы. Переделай этот кусок и исправь другие места